package/libpiksi/libpiksi/src/settings_daemon.c

```diff
@@ -0,0 +1,551 @@
+/*
```

Comment:
I'm definitely not a fan of `settings_daemon.c` for this file. I realize this is client code for interacting with the `settings_daemon` but it just feels too specific. we can keep as `settings.c` or maybe `settings_client.c`